packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.types.ts

```diff
@@ -108,7 +108,7 @@ export interface IDetailsRowBaseProps extends IBaseProps<IDetailsRow>, IDetailsI
   /**
    * Callback for rendering an item column
    */
-  onRenderItemColumn?: (item?: any, index?: number, column?: IColumn) => any;
+  onRenderItemColumn?: (item?: any, index?: number, column?: IColumn) => React.ReactNode;
```

Comment:
We could use a lookup type here @JasonGore, what do you think?

Reply:
Pretty certain `React.ReactNode` is the correct type here.

Reply:
Is it redundant against `IDetailsListProps`? I think a `Pick` might be better here. I tried this locally and it seemed to work:
```tsx
export interface IDetailsRowBaseProps extends Pick<IDetailsListProps, 'onRenderItemColumn'>, IBaseProps<IDetailsRow>, IDetailsItemProps {
  ...
}
```

Reply:
Using `Pick` to keep the instances of `onRenderItemColumn` in sync. Will follow-up separately with the remainder.